docker/qchem.dockerfile

```diff
@@ -1,31 +0,0 @@
-# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
```

Comment:
What is the purpose of removing these docker files? Is this related to pennylane-qchem installation?

Reply:
Yep I think it is fine to remove. Although, it might be worth adding pyscf and openfermion to the main PL docker image?

Reply:
Added `pip install openfermionpyscf` to pennylane.dockerfile. It installs openfermion and pyscf as well.